README.md

```diff
@@ -46,3 +46,16 @@ Contact us:
 # License
 
 [MIT License](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/blob/master/LICENSE) (open source)
+
+# Notice
+
+The TM:PE team is happy to support you if you have any issues with the mod under the following conditions:
+- You are using the latest version of the stable and/or labs mod.
```

Comment:
Capitalisation of LABS and STABLE for consistency with mod title in workshop?

Reply:
Will do when I get the chance 👍